### ISSUE TITLE
getExtent (tree-test.js) is incorrect

### ISSUE THREAD

**Issue by vird** [tree-sitter/tree-sitter#2893]
https://github.com/tree-sitter/tree-sitter/blob/1d8975319c2d5de1bf710e7e21db25b0eee4bc66/lib/binding_web/test/tree-test.js#L414-L422
```
console.log(getExtent("123\n456"))
// {row: 2, column: 8}
```
column is incorrect

```
console.log(getExtent(""))
// {row: 1, column: 1}
```
row and column should start from 0 (I'm not sure, but at least I get { row: 0, column: 0 } as rootNode.startPosition)

**Comment by amaanq** [comment]
Yeah, `getExtent` gets the position at the end of the given text, and is only used on single line strings in tests - multi-line strings probably weren't considered, hence why the column looks incorrect, but that should be fixed
